### PR TITLE
Perform yaml-sort on i18n pre-commit

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
**Description:**

This PR adds a pre-commit hook to execute yaml-sort on modified language files prior to commit.
yaml-sort will sort keys, reformat some content and strip comments, so I moved comments to Weblate for folks who contribute translations.

Note: non-language YAML files (e.g. config files) are not affected by this PR.

**PR Checklist:**
- [na] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [na] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks great!

**Review by miles-grant-ibigroup — APPROVED** [review]
Great improvement, thanks!

File: README.md [hunk 1]
```diff
@@ -62,6 +111,7 @@ OTP-react-redux now uses [Hosted Weblate](https://www.weblate.org) to manage tra
 Translations from the community are welcome and very much appreciated,
 please see instructions at https://hosted.weblate.org/projects/otp-react-redux/.
 Community input from Weblate will appear as pull requests with changes to files in the `i18n` folder for our review.
+(We reserve the right to edit or reject contributions as we see fit.)
```
[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply README.md:114]: I rejected a few ones to the English file in the past (editing was left enabled by error) and might reject some for French that we got back.

[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply README.md:114]: I'm still not a huge fan of the "we" here -- could we change to something like "contributions may be edited or rejected to remain in line with long-term project goals"?

[the remark this answers is not in the document]
Comment (AdrianaCeric) [reply README.md:114]: ^ Agree with removing "we". If changes are made here, don't forget to change [this line](https://github.com/opentripplanner/otp-ui/pull/562/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R151) too

[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply README.md]: Good rephrasing suggestion, updated in https://github.com/opentripplanner/otp-react-redux/pull/828/commits/b18cfffdf03051105dfe506fbae1834c2b93f9b3.

[the remark this answers is not in the document]
Comment (binh-dam-ibigroup) [reply README.md]: I also updated the OTP-UI README, see https://github.com/opentripplanner/otp-ui/pull/562/commits/10ff452dc3ccf529517406c3604508abcf1234a8.

**Pull request merged**